pkg/minikube/reason/reason.go

```diff
@@ -460,4 +460,25 @@ var (
 		`),
 		Style: style.SeeNoEvil,
 	}
+
+	NotFoundCriDockerd = Kind{
+		ID:       "NOT_FOUND_CRI_DOCKERD",
+		ExitCode: ExProgramNotFound,
+		Advice: translate.T(`The none driver with Kubernetes v1.24+ requires cri-dockerd.
```

Comment:
We should also mention only when using the `docker` container-runtime.

Reply:
Appreciate the suggestion, updated with:

`The none driver with Kubernetes v1.24+ and the docker container-runtime requires cri-dockerd.`